src/components/RenderHTML.js

```diff
@@ -65,11 +68,24 @@ function CodeRenderer({
     // We split wrapper and inner styles
     // "boxModelStyle" corresponds to border, margin, padding and backgroundColor
     const {boxModelStyle, otherStyle: textStyle} = splitBoxModelStyle(style);
+
+    const italic = textStyle.fontStyle === 'italic' && fontFamily.MONOSPACE_ITALIC;
+    const bold = textStyle.fontWeight === 'bold' && fontFamily.MONOSPACE_BOLD;
+    const italicBold = italic && bold && fontFamily.MONOSPACE_BOLD_ITALIC;
+
+    const font = italicBold || bold || italic || fontFamily.MONOSPACE;
+
+    const textStyleOverride = {
+        fontFamily: font,
+        fontWeight: undefined,
+        fontStyle: undefined,
+    };
+
     return (
         <InlineCodeBlock
             TDefaultRenderer={TDefaultRenderer}
             boxModelStyle={boxModelStyle}
-            textStyle={textStyle}
+            textStyle={{...textStyle, ...textStyleOverride}}
```

Comment:
We should pass the new styles as Array as per the new StyleGuide. Change the prop type if necessary. Thanks.

Reply:
🤔  I take a look and what this component expect is an object